app/src/test/java/com/losandroides/learn/domain/ItemMapperKtTest.kt

```diff
@@ -0,0 +1,36 @@
+package com.losandroides.learn.domain
+
+import com.losandroides.learn.data.network.item.model.ItemDTO
+import com.losandroides.learn.data.network.item.model.ItemsDTO
+import com.losandroides.learn.domain.model.Item
+import org.amshove.kluent.shouldBeEqualTo
+import org.junit.Test
+
+class ItemMapperKtTest {
+
+    @Test
+    fun `GIVEN empty DTO list WHEN ItemsDTO toDomain THEN return the expected domain list`() {
+        val itemsDTO = ItemsDTO(emptyList())
+        val items = emptyList<Item>()
```

Comment:
Very purist comment, this `items` belongs to the `Arrange/Assert` section, and where you put it is in the `Given` section. 

So for instance you could do `result shouldBeEqualTo emptyList<Items>()` but it's just a comment, nothing important :P

Reply:
Good catch!
You're right.
I'll change it! Thanks, bro! :slightly_smiling_face: